react/features/toolbox/components/native/HangupMenu.js

```diff
@@ -0,0 +1,81 @@
+// @flow
```

Comment:
We are using TS for all new components, can you please adjust? It should be reasonably straightforward now.

Reply:
Done